lib/src/day_view/day_view.dart

```diff
@@ -634,7 +634,7 @@ class DayViewState<T extends Object?> extends State<DayView<T>> {
                     date.month,
                     date.day,
                     0,
-                    minuteSlotSize.minutes * i,
+                    minuteSlotSize.minutes * i + (_startHour * 60),
```

Comment:
This appears to be a duplicate widget in both the day view and the week view. Could you please create a separate, common widget for this?